src/ontology/uberon-edit.obo

```diff
@@ -113928,7 +113926,6 @@ xref: SCTID:276832000
 xref: UMLS:C0022651 {source="ncithesaurus:Renal_Calyx"}
 is_a: UBERON:0000063 ! organ subunit
 relationship: contributes_to_morphology_of UBERON:0001224 ! renal pelvis
-relationship: part_of UBERON:0001224 ! renal pelvis
```

Comment:
I think this should be in a separate PR, as it's a distinct issue and potentially controversial.  ASCT+B tables have this link as well as Uberon

<img width="248" alt="image" src="https://user-images.githubusercontent.com/112839/157035486-60815903-f08f-46d3-8441-f0cd6ef96810.png">
See https://hubmapconsortium.github.io/ccf-asct-reporter/vis?selectedOrgans=kidney-v1.1&playground=false

Reply:
Makes sense. Added back:
 'kidney calyx' part_of some 'renal pelvis'.